sites/star.azure.zooniverse.org.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    include /etc/nginx/ssl.default.conf;
+    server_name "~^(?P<subdomain>.*)\.azure\.zooniverse\.org$";
+
+    location / {
+        resolver 8.8.8.8;
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
+        proxy_http_version 1.1;
+
+        # This should point to static.zooniverse.org after the full DNS switch is complete
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|woff|ttf|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|tar|gz|tgz|bz2|tbz2|zip|js|css)$ https://zooniversestatic.z13.web.core.windows.net/$subdomain$request_uri;
```

Comment:
this subdomain may need to be expanded to include the `.zooniverse.org` domain path to hit the correct azure blob paths in the `$web` container, see my other review comment below
```
https://zooniversestatic.z13.web.core.windows.net/$subdomain.zooniverse.org$request_uri;
```

Reply:
yup, that sounds right